node/src/indexer/handler.rs

```diff
@@ -75,15 +75,14 @@ async fn handle_message(
     mpc_contract_id: &AccountId,
     _account_public_key: &Option<PublicKey>,
     sign_request_sender: mpsc::UnboundedSender<ChainSignatureRequest>,
-    indexer_state: Arc<IndexerState>,
+    _indexer_state: Arc<IndexerState>,
```

Comment:
should we just remove the two unused fields? Do you see them being used again in the near future?